tests/MongoDB.Driver.Core.Tests/Core/Operations/EndTransactionOperationTests.cs

```diff
@@ -117,22 +125,49 @@ public void CreateCommand_should_return_expected_result_when_write_concern_is_se
             result.Should().Be("{ fakeOperation : 1 }");
         }
 
+        [Fact]
+        public void RecoveryToken_should_return_expected_result()
```

Comment:
If we use reflection this test would not be necessary.

Reply:
Using reflection so I'm yanking the test per your suggestion :)